cfpb/cfgov-refresh
# Upgrade chart-builder to v3.7.0 for inquiry index + credit tightness charts

We'll be pushing at least one more upgrade before launch next week, but we need to get this one onto build and beta now so we can review the inquiry index and credit tightness charts staged there before launch on Monday.

## Changes

- Upgrade to [latest version of cfpb-chart-builder](https://github.com/cfpb/cfpb-chart-builder/releases/tag/v3.7.0) with projected data changes 

## Testing

1. `./frontend.sh` should install new version of chart builder + build assets
1. `./runserver.sh`
1. log into `http://localhost:8000/admin`
1. create a new page with 2 Chart Blocks with type LineIndex or add to existing page, with the following data sources:
 - denials/credit tightness: https://files.consumerfinance.gov/data/consumer-credit-trends/auto-loans/den_data_AUT.json
 - inquiry index https://files.consumerfinance.gov/data/consumer-credit-trends/auto-loans/inq_data_AUT.json
1. confirm rapsberry colored charts load and show 4 months of projected data for the inquiry index chart and 0 projected data for credit tightness


## Screenshots


## Notes

- **KNOWN ISSUE** - there is a known bug in Chrome 69 and Firefox 62 where the "Values after __ are projected" label is not visible. This will be fixed in a future version of chart-builder before launch. We need to still see the charts on staging before then, which is why I'm submitting this PR now without that fix.

## Todos

-

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
